webpack/src/ForemanDiscovery/DiscoveredHosts/Components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions
+} from "@patternfly/react-core";
+import { CubesIcon } from "@patternfly/react-icons";
+
+const HostEmptyState = () => (
+  <EmptyState variant={EmptyStateVariant.small}>
```

Comment:
We have a foreman-core component that should give you the same empty-state-pattern, but using pf3:
https://github.com/theforeman/foreman/tree/develop/webpack/assets/javascripts/react_app/components/common/EmptyState

I am wondering if we should migrate our current empty-state-component in foreman-core and reuse it here so all the other pages would enjoy the same pf4 design and it would be consistent over core and plugins.

Reply:
https://foreman-storybook.surge.sh/?path=/docs/components-empty-state-pattern--default-story

Reply:
This sounds interesting, let me try to open a PR in foreman core for this.